issue-rules.yml

```diff
@@ -308,6 +323,11 @@ rules:
   contains: 'PublishCodeCoverageResults'
   addLabels: ['Area: CodeCoverage']
 
+# Area: ABTT
```

Comment:
Can we group all the ABTT tasks into this section?

Reply:
Sure, I moved it into one section